plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraRecordCursor.java

```diff
@@ -24,21 +25,26 @@
 
 import java.util.List;
 
+import static com.google.common.base.Preconditions.checkArgument;
```

Comment:
This change one moving from an index based reading data to a columnName based - can be moved to a different commit. WDYT ?